src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -268,6 +314,33 @@ class ZHAConfigDashboard extends LitElement {
     this._networkSettings = await fetchZHANetworkSettings(this.hass!);
   }
 
+  private async _fetchDevices() {
+    try {
+      this._devices = await fetchDevices(this.hass!);
+    } finally {
+      this._updateStatus();
+    }
```

Comment:
Please also add a catch and an error state variable to show a `ha-alert` when an error occurs.